app/modules/node.c

```diff
@@ -305,6 +305,7 @@ static int node_key( lua_State* L )
 #endif
 
 extern lua_Load gLoad;
+extern bool user_process_input(bool force);
```

Comment:
So would I, but this should logically be `user_main.h` which doesn't exist and I don't want to start adding this include file in this commit.  I added the extern declaration here because it wasn't previously declared as external at all, and ths practice is used elsewhere in the firmware for such odd externs.  A job for another day.